artifacts/deploy/bootstrap-token-configuration.yaml

```diff
@@ -140,7 +131,6 @@ rules:
   - resourceinterpreterwebhookconfigurations
   - resourceinterpretercustomizations
   verbs:
-  - get
```

Comment:
Get is the read permission for a single object, and list is the read permission for objects. Since list is necessary, get permission can also be retained.